buildSrc/src/main/groovy/com/carrotsearch/gradle/junit4/TestProgressLogger.groovy

```diff
@@ -49,21 +53,81 @@ class TestProgressLogger implements AggregatedEventListener {
         progressLogger = factory.newOperation(TestProgressLogger)
         progressLogger.setDescription('Randomized test runner')
         progressLogger.started()
-        progressLogger.progress('Starting JUnit4 with ' + e.getSlaveCount() + ' jvms')
+        progressLogger.progress("Starting JUnit4 with $e.slaveCount jvms")
```

Comment:
I've been trying to stick with the pattern of always using `${}` for clarity of what is groovy and what is string.